### ISSUE TITLE
Blank Neo4j Desktop after uninstall and re-installation (Windows).

### ISSUE THREAD

**Issue by AltheimIcarus** [neo4j-apps/neo4j-desktop#43]
For my case, simply unintall existing Neo4j desktop then reinstall it again. Neo4j Desktop then startup fine but gone blank after the loading screen.
Fixed this by deleting the remaining files after uninstallation in 

    C:/Users/<username>/.neo4j 

and 

    C:/Users/<username>/.Neo4jDesktop 

before performing clean installation. Now Neo4j Desktop can run without error.

**Comment by AltheimIcarus** [comment]
The version of Neo4j Desktop is 1.1.13.

**Comment by Metaur** [comment]
@AltheimIcarus have you got log file saved before wiping out `.Neo4jDesktop` folder?

**Comment by AltheimIcarus** [comment]
@Metaur Nope, I wiped the whole directory. I did save a copy somewhere else as old.Neo4jDesktop. Besides, I tried replacing newly created .Neo4jDesktop with the data in the old.Neo4jDesktop. Somehow it worked like restoring a backup.

**Comment by serafdev** [comment]
I got the same issue for some reason, I'm using MacOS Catalina, not sure where I can find the `.Neo4jDesktop` or the logs, I searched using `find` for the dotfile and looked into `/var/log` for the logs.

Neo4j Desktop v1.3.8:

![image](https://user-images.githubusercontent.com/16715599/94483889-95d81800-01a9-11eb-91c1-4b3f182721f2.png)
